cloudberry/neo/conf/application.conf

```diff
@@ -93,3 +93,5 @@ asterixdb.url = "http://localhost:19002/query/service"
 asterixdb.lang = SQLPP
 #asterixdb.lang = sparksql
 asterixdb.view.meta.name = "viewMeta"
+
+play.websocket.buffer.limit = 5M
```

Comment:
Actually, for TwitterMap example, 5M might be a bit too much. I just put a safe value for future cases.